_data/accessibility-tests/icon.yml

```diff
@@ -0,0 +1,50 @@
+title: Icon
+component_status: pass
+test_items:
+# General items
+  - summary: Color is not the only method used to convey icon information.
+    summary_additional: |
+      When you view an icon that represents an action and determine the action,
+      the text, and never color alone, conveys the icon's meaning where necessary.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.1
+  - summary: Icons meet color contrast requirements.
+    summary_additional: |
+      When you view the icon on a page
+      and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the icon and background is at least 3:1.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.11
+  - summary: Each icon has a consistent meaning across pages.
+    summary_additional: |
+      When you use a screen reader or view the page
+      and determine the purpose of the icon,
```

Comment:
I suggest deleting "and determine the purpose of the page" here

Reply:
Updated in [abe962f](https://github.com/uswds/uswds-site/pull/2518/commits/abe962f72b0e14d17d23ecf9edbb4e406da5e570)